code/game/gamemodes/meteor/meteors.dm

```diff
@@ -137,6 +137,7 @@
 	//first bust whatever is in the turf
 	for(var/atom/A in T)
 		if(A != src)
+			A.visible_message("<span class='warning'>[src] slams into [A].</span>", "<span class='userdanger'>[src] slams into you!.</span>")
```

Comment:
mob/visible_message() and atom/proc/visible_message() do not take the same arguments and in the same order.